code/modules/detectivework/tools/samplekit.dm

```diff
@@ -0,0 +1,72 @@
+/obj/item/forensics
+	icon = 'icons/obj/forensics.dmi'
+	flags = NOBLUDGEON
+
+/obj/item/forensics/proc/handlefiber(ftext)
+	return replacetext(replacetext(replacetext(ftext, "material from a pair of ", ""), ".", ""), "Fibers from a", "")
```

Comment:
Replace `\(material from a pair of |\.|Fibers from a)\g` in ftext with empty string.
If the space after `material from a pair of` is optional: `\(material from a pair of ?|\.|Fibers from a)\g`
If in case of there  being space after `Fibers from a` it should be removed: `\(material from a pair of ?|\.|Fibers from a ?)\g`